acceptance/help_test.go

```diff
@@ -22,6 +22,7 @@ Commands:
   fetch                   fetches releases
   find-release-version    prints a json string of a remote release satisfying the Kilnfile version and stemcell constraints
   help                    prints this usage information
+  pre-process             preprocess yaml files
```

Comment:
I'm still unsure what pre-process means in this context

Reply:
For future readers (Carlos and I talked on zoom), the code kind-of-simply runs the config files through [Go standard library's  template library](https://golang.org/pkg/text/template/). We use this internally on the Platform Provider Team to do conditional logic for a set of config files that are used to generate two different tiles. The "tiles" have different instance group colocation strategies (Tanzu Application Service for VMs small footprint (srt) vs. the full tiles). We have been doing this for a long time and are trying to reduce the complexity of what it takes to build our tiles. So, we are adding proposing kiln has this feature as a first class feature. In the future, I would like kiln bake to handle templated configuration files gracefully.